pom.xml

```diff
@@ -163,13 +163,12 @@
                     <imageManagement>
                         <image>
                             <repository>${dockerHubPublic}/cafapi/opensuse-jre17</repository>
-                            <tag>4.0.0</tag>
-                            <digest>sha256:c0065ce7db226f864099ffd52371ebcf580c944a772f24fc0cb69ccacd9e6059</digest>
+                            <tag>4.0.1</tag>
+                            <digest>sha256:434c16f975fbbe2f4adfc879d88a32b5f8fcc92a2d8e29e3598dc10088717405</digest>
                         </image>
                         <image>
                             <repository>${dockerHubPublic}/opensearchproject/opensearch</repository>
                             <tag>1.3.16</tag>
-                            <digest>sha256:a0a5876aab97a0137e9fe21f8856e6a6b93c873bb97e13b6a960c0b20e33a711</digest>
```

Comment:
@dermot-hardy I don't think the opensearch images are static, when I pull locally I get the sha above but when I look in dockerhub the sha is different and the docker plugin also finds a different sha. I note that OS 1.3.16 was released on April 23rd but the image was updated 22 days ago so that doesn't tally.

Reply:
Would not be shocked if you're right but drop it from this for now until we find out for sure.

Reply:
Would not be shocked if you're right but drop it from this for now until we find out for sure.